tests/client/amo/test_utils.js

```diff
@@ -1,15 +1,50 @@
 import createStore from 'amo/store';
+import * as featuredActions from 'amo/actions/featured';
 import * as landingActions from 'amo/actions/landing';
 import * as api from 'core/api';
 import {
+  ADDON_TYPE_EXTENSION,
   ADDON_TYPE_THEME,
   SEARCH_SORT_POPULAR,
   SEARCH_SORT_TOP_RATED,
 } from 'core/constants';
-import { loadLandingAddons } from 'amo/utils';
+import { loadFeaturedAddons, loadLandingAddons } from 'amo/utils';
 
 
 describe('amo/utils', () => {
+  describe('loadFeaturedAddons()', () => {
+    const addonType = ADDON_TYPE_EXTENSION;
+    let ownProps;
+
+    beforeEach(() => {
+      ownProps = {
```

Comment:
It doesn't look like this is shared by multiple tests anymore. I suggest moving it closer to the source for easier readability until it needs to be shared.